components/teams-service/server/v2/server_test.go

```diff
@@ -1046,18 +1041,25 @@ func runAllServerTests(
 			cleanupTeamV2(t, cl, createTeam.Team.Id)
 		})
 
-		t.Run("when team does not exist, returns Not Found error", func(t *testing.T) {
+		t.Run("when no user ids provided, returns invalid request", func(t *testing.T) {
```

Comment:
do we have another `when team does not exist` test somewhere else?

Reply:
blerg this was def an accident thx!